src/control/server/mgmt_pool.go

```diff
@@ -30,13 +30,9 @@ const (
 	// DefaultPoolNvmeRatio defines the default NVMe:SCM ratio for
 	// requests that do not specify one.
 	DefaultPoolNvmeRatio = 0.94
-	// DefaultPoolServiceReps defines a default value for pool create
-	// requests that do not specify a value. If there are fewer than this
-	// number of ranks available, then the default falls back to 1.
-	DefaultPoolServiceReps = 3
 	// MaxPoolServiceReps defines the maximum number of pool service
 	// replicas that may be configured when creating a pool.
-	MaxPoolServiceReps = 13
+	MaxPoolServiceReps = 2*daos.PoolSvcRedunFacMax + 1
```

Comment:
[Will this cause some interoperability problem if some crazy users use 13 service reps before upgrading to this specific version?

Reply:
This constant is only used when creating a pool. Your concern might apply to the C upgrade code, however, for I think I didn't cap the PS RF converted from the number of replicas. (I tested upgrading from release/2.2 with at most 5 replicas only, but not more than 9.)

Reply:
Taken care of.